packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.test.tsx

```diff
@@ -0,0 +1,123 @@
+import { MessageType } from '@quiet/state-manager'
+import React from 'react'
+
+import { renderComponent } from '../../../testUtils/renderComponent'
+import NestedMessageContent from './NestedMessageContent'
+
+describe('NestedMessageContent', () => {
+  it('renders message', () => {
+    const message = {
```

Comment:
Guess you can make this object global and extend it with media param in the last test